### ISSUE TITLE
progress bar not animating in vue js

### ISSUE THREAD

**Issue by obonyojimmy** [kimmobrunfeldt/progressbar.js#124]
i have succefully insalled and run progressbar js to my vue.js app , however its not animating despite calling animate option , what might be the problem below in my componen.vue code 

```
<style>
	
</style>

<template>

<div class="" id="{{ randomid }}"></div>
</template>

<script>

var ProgressBarJs = require('progressbar.js')

module.exports = {
   props: {
     options: {
      type: Object,
      default :function () {
       return {
		  strokeWidth: 4,
		  easing: 'easeInOut',
		  duration: 1400,
		  color: '#FFEA82',
		  trailColor: '#eee',
		  trailWidth: 1,
		  svgStyle: {width: '100%', height: '100%'}
       }
      }
    },
    animate: {
      type: Object,
      default :function () {
       return {
         progress : 0.7 , //Number from 0.0 to 1.0
         
       }
      }
    },
    type: {
      type: String,
      default: 'Line' 
      
    }
    },
	created: function() {
	   
	   
	},
	ready: function(){
	    this.drawProgressbarjs();
	   
	},
	methods : {
	        
        drawProgressbarjs: function () {
           var line , circle ;
           
           if(this.type == 'Line' || this.type === 'Line' ){
		     line = new ProgressBarJs.Line('#'+this.randomid,this.options);
		     line.animate(this.animate.progress);  // Number from 0.0 to 1.0
		   }
		   
		   if(this.type == 'Circle' || this.type === 'Circle' ){
		   
		    circle = new ProgressBarJs.Circle('#'+this.randomid, this.options );

			circle.animate(this.animate.progress, function() {
				circle.animate(0);
			})
		     
		   }
		   
	       
	      
	    
	    }
	    
		
	
	},
	computed: {
    randomid: function () {
     return _.uniqueId('progressbar-js_')
    }
  }

}
</script>
```

**Comment by kimmobrunfeldt** [comment]
I'm sorry but your code is too hard to follow. Please clean up the code indentation and senseless if clauses like this: ` if(this.type == 'Circle' || this.type === 'Circle' ){`. 

Your approach seems right: generate a random id for each instance of progress bar.

**Comment by khizerhassan** [comment]
Facing this issue right now. :(
